### PR TITLE
added ability to cast to appropriate types within the model

### PR DESCRIPTION
**Pull request by elorest**
so you can just send it string values and don't have to do conversions in the controller.

```cr
p = Post.new
# works with namedtuple
p.set_attributes(title: "auto casting saves time", description: "I love this feature",  tag_id: 35)
# or hash
p.set_attributes({:title => "auto casting saves time", :description => "I love this feature",  :tag_id => 35})
```

I played around with tests but there don't seem to be any existing ones that work for this object. If you could get a couple base things working for testing the model I'll add some for this.

### CODE REVIEW

**Comment by drujensen** [comment]
this is awesome!

So you can do something like this?
```
p.set_attributes(params.select { |key| %w(name description tag_id).include? key })
```

File: src/kemalyst-model.cr [hunk 1]
```diff
@@ -98,6 +98,28 @@ class Kemalyst::Model
         {% end %}
         return fields
     end
+    
+    # Cast params and set fields.
+    private def cast_to_field(name, value)
+      case name.to_s
+        {% for n, type in FIELDS %}
+        when "{{ n.id }}"
+          {% if type.id == Int32.id %}
```
Comment (drujensen) [inline src/kemalyst-model.cr]: Can you add Int64?

**Comment by elorest** [comment]
Int64 done.

**Comment by drujensen** [comment]
released v0.5.6

**Comment by elorest** [comment]
Why do we use `will/crystal-pg` instead of the default crystal one?

**Comment by drujensen** [comment]
did it move to crystal-lang?

**Comment by elorest** [comment]
Nevermind. I guess it didn't. I wonder why not.

**Pull request merged**